src/main/java/urjc/isi/dao/implementaciones/PeliculasDAOImpl.java

```diff
@@ -131,6 +131,17 @@ public void deleteByID(String idpelicula){
 				case "duracion":
 					cond+= "p.duracion>"+"'"+conditions.get("duracion")+"'";
 					break;
+				case "titulo":
+					cond+= "p.titulo like "+"'"+conditions.get("titulo")+"%'";
```

Comment:
¿Solo queréis lo que rellene lo que viene detrás de lo que ponga el usuario?

Reply:
No entiendo la pregunta.

Reply:
Que vuestra aplicación añadiendo Harry te lista te listaría todas las de Harry Potter, pero poniendo solamente Potter no, porque no hay ninguna que empiece por Potter, ¿es ese el comportamiento que buscáis?

Reply:
Claro, así lo especifiqué tanto en la issue como en la wiki, lo que queremos es coger las películas que "empiezan por x". Más tipos de filtrado ("contiene la palabra x", "el título exacto es x") se podrán añadir más adelante.